### PR TITLE
connect search frontend to api and made food results component

### PR DESCRIPTION
**Pull request by olukukoyi**
connect /search-food component to its respective API

when users navigate to `/search-food` they can input a food name in the search button at the top and the user will be redirected to `/search-food/<food_name_by_user_input>` to see a list of the results

from there, users can select a food they want to see the details of.

I also:
* added test,
* set up navigation for the sidebar ( so when user clicks `add food` , they navigate to `/search-food`

Later, I will redirect the user to `/food-item` so the user can view the details of the respective food.

**Question**: do I have to test if it renders food items correctly? -> If it clicks a food from search-food results and goes to food item to see the details

### CODE REVIEW

**Comment by Abdul-Andha** [comment]
To answer your question: testing if the correct food is rendered would be useful 🙂

**Comment by olukukoyi** [comment]
> Is this the intended styling?
> 
> image

yea, until wireframe is made

File: topics-project/__e2e__/searchfoods.spec.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
+// @ts-check
+
+import { test, expect } from "@playwright/test";
+
+test("component renders correctly", async ({ page }) => {
```
Comment (Abdul-Andha) [inline topics-project/__e2e__/searchfoods.spec.js:5]: is this an e2e test? Seems more like a unit test

File: topics-project/app/(main)/search-food/components/IndividualFoodItem.jsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            <path
+              fill="currentColor"
+              d="M834.1 469.2A347.5 347.5 0 0 0 751.2 354l-29.1-26.7a8.09 8.09 0 0 0-13 3.3l-13 37.3c-8.1 23.4-23 47.3-44.1 70.8c-1.4 1.5-3 1.9-4.1 2s-2.8-.1-4.3-1.5c-1.4-1.2-2.1-3-2-4.8c3.7-60.2-14.3-128.1-53.7-202C555.3 171 510 123.1 453.4 89.7l-41.3-24.3c-5.4-3.2-12.3 1-12 7.3l2.2 48c1.5 32.8-2.3 61.8-11.3 85.9c-11 29.5-26.8 56.9-47 81.5a295.6 295.6 0 0 1-47.5 46.1a352.6 352.6 0 0 0-100.3 121.5A347.75 347.75 0 0 0 160 610c0 47.2 9.3 92.9 27.7 136a349.4 349.4 0 0 0 75.5 110.9c32.4 32 70 57.2 111.9 74.7C418.5 949.8 464.5 959 512 959s93.5-9.2 136.9-27.3A348.6 348.6 0 0 0 760.8 857c32.4-32 57.8-69.4 75.5-110.9a344.2 344.2 0 0 0 27.7-136c0-48.8-10-96.2-29.9-140.9M713 808.5c-53.7 53.2-125 82.4-201 82.4s-147.3-29.2-201-82.4c-53.5-53.1-83-123.5-83-198.4c0-43.5 9.8-85.2 29.1-124c18.8-37.9 46.8-71.8 80.8-97.9a349.6 349.6 0 0 0 58.6-56.8c25-30.5 44.6-64.5 58.2-101a240 240 0 0 0 12.1-46.5c24.1 22.2 44.3 49 61.2 80.4c33.4 62.6 48.8 118.3 45.8 165.7a74.01 74.01 0 0 0 24.4 59.8a73.36 73.36 0 0 0 53.4 18.8c19.7-1 37.8-9.7 51-24.4c13.3-14.9 24.8-30.1 34.4-45.6c14 17.9 25.7 37.4 35 58.4c15.9 35.8 24 73.9 24 113.1c0 74.9-29.5 145.4-83 198.4"
```
Comment (Abdul-Andha) [inline topics-project/app/(main)/search-food/components/IndividualFoodItem.jsx:62]: Is there a way to make this more readable? Maybe make the svg in a different file and import it

File: topics-project/app/api/search-foods/[food_name]/route.js [hunk 3]
```diff
@@ -15,9 +15,12 @@ export async function GET(request, context) {
     for (const food of foods) {
       let nutrients = getfoodNutrients(food.foodNutrients); // grabs all the nutrients
 
+      // console.log(food.packageWeight);
```
Comment (Abdul-Andha) [inline topics-project/app/api/search-foods/[food_name]/route.js:18]: delete this clg

**Review by Abdul-Andha — COMMENTED** [review]
PR looks good. I just dropped some questions

File: topics-project/app/(main)/search-food/components/IndividualFoodItem.jsx (hunk 2 above)
Comment (JohanDelao) [reply topics-project/app/(main)/search-food/components/IndividualFoodItem.jsx:62]: replace svg with this https://lucide.dev/icons/flame

**Pull request merged**